### ISSUE TITLE
UTC hours auto-incrementing on date change

### ISSUE THREAD

**Issue by Zertsov** [Vuepic/vue-datepicker#182]
When switching a date while using `utc="preserve"`, any subsequent date selection will increment the hour by 1, even if you select the exact same date. Reproducable in [StackBlitz](https://stackblitz.com/edit/vuepic-vue-datepicker-szxecf?file=src/components/Playground.vue) and locally with the same code. This shouldn't increment at all when the confirm button is clicked.
Happens with and without `range` prop as well.

```vue
<template>
  <div>
    <Datepicker v-model="date" placeholder="blah" utc="preserve" />
  </div>
</template>

<script setup>
import { ref } from 'vue';
import Datepicker from '@vuepic/vue-datepicker';
import '@vuepic/vue-datepicker/dist/main.css';

const date = ref(new Date()); // also happens with all versions of Date()
</script>
```

**Comment by Zertsov** [comment]
Further, this seems to only happen when confirming. In the `internal-model-change` emitter the times are correct.

**Comment by Zertsov** [comment]
PR with fix: https://github.com/Vuepic/vue-datepicker/pull/184

**Comment by Jasenkoo** [comment]
Will close when the version is published 😄

**Comment by Zertsov** [comment]
> Will close when the version is published 😄

My bad! Thanks for the review and merge!

**Comment by Jasenkoo** [comment]
Fix is available under [`3.6.0-alpha.1`](https://www.npmjs.com/package/@vuepic/vue-datepicker/v/3.6.0-alpha.1)
